src/test/java/test/integration/ContainerInstallationIntegrationTest.java

```diff
@@ -268,7 +301,7 @@ private void installContainerAssertProfileAndDependencies(final String container
     }
 
     private void executeCmd(String container) throws TimeoutException {
-        shell().execute("arquillian-setup --container-adapter " + container + " --test-framework junit", 300);
+        shell().execute("arquillian-setup --container-adapter " + container + " --test-framework junit", 15);
```

Comment:
yeah but we are using by default value as 15 in [ShellExecutor](https://github.com/forge/arquillian-addon/blob/master/src/test/java/test/integration/support/ShellExecutor.java#L24). So I think we should increase it as well.